virttest/utils_libvirt/libvirt_disk.py

```diff
@@ -790,10 +791,16 @@ def check_virtual_disk_io(vm, partition, path="/dev/%s"):
         cmd = (
             "fdisk -l && mkfs.ext4 -F {0} && "
             "mkdir -p test && mount {0} test && echo"
-            " teststring > test/testfile && umount test".format(path % partition)
+            " teststring > test/testfile".format(path % partition)
         )
         status, output = session.cmd_status_output(cmd)
         LOG.debug("Disk operation in VM:\nexit code:\n%s\noutput:\n%s", status, output)
+        if umount:
+            umount_cmd = "umount test"
+            status, output = session.cmd_status_output(umount_cmd)
```

Comment:
The value of 'status' will be overwritten here, the result of command (as below) in  L792-795 will never be checked. Should you check the 'mount' commands results first and then check result of 'umount'?
```
 cmd = (
            "fdisk -l && mkfs.ext4 -F {0} && "
            "mkdir -p test && mount {0} test && echo"
            " teststring > test/testfile && umount test".format(path % partition)
```

Reply:
Thanks for your useful suggestion. I directly use the different names, for example, u_status and u_output, but not the same one now.